tests/test_types.py

```diff
@@ -733,6 +734,34 @@ class Model(BaseModel):
     ]
 
 
+def test_str_limited_good():
+    max_length = 5
+
+    class Model(BaseModel):
+        v: StrLimited[max_length]
+
+    m = Model(v='abcde')
+    assert m.v == 'abcde'
+
+
+def test_str_limited_too_long():
+    max_length = 5
+
+    class Model(BaseModel):
+        v: StrLimited[max_length]
```

Comment:
```suggestion
        v: StrLimited[5]
```
same, always best to hard code tests unless you need to vary the value, which you don't here.